bin/deploy_to_dockerhub

```diff
@@ -6,5 +6,5 @@ username=$2
 password=$3
 
 docker login -u "$username" -p "$password"
-bin/api_package_docker "$tag"
-bin/dashboard_package_docker "$tag"
+bin/api_package_docker "$tag" latest
```

Comment:
This does not do what you think it does :) you simply build docker image with $tag here, but with "latest" only override $tag - not create additional one. So I suggest a bit of refactoring to these scripts. for example to have script push_to_githbut "docker_image_name" [list of tags to push]